bool excludePlane
int16 numberOfPlanes
----
sensor_msgs/PointCloud2[] clouds
geometry_msgs/Quaternion[] coefficients
